Feat/cloud task scripts (#4236)
influxdata/docs-v2: content/influxdb/v2.3/process-data/get-started.md

The InfluxDB UI provides a form for defining task options.
{{% /note %}}

{{% cloud-only %}}

### Task options for invokable scripts

> With the InfluxDB Cloud API, you can create tasks that reference and run [invokable scripts](influxdb/cloud/api-guide/api-invokable-scripts/).
> When you create or update the task, you pass task options as properties in the request body--for example:
> 

Reviewer: ```suggestion
Use the InfluxDB Cloud API to create tasks that reference and run [invokable scripts](influxdb/cloud/api-guide/api-invokable-scripts/).
When you create or update the task, pass task options as properties in the request body--for example:

```

_See [Common tasks](/influxdb/v2.3/process-data/common-tasks) for examples of tasks commonly used with InfluxDB._

{{% cloud-only %}}

> ### Process data with invokable scripts

Reviewer: This sections feels like it should be in the "Create a task to run an invokable scripts" doc rather than in the "Get started with InfluxDB tasks" doc.

Author: > This sections feels like it should be in the "Create a task to run an invokable scripts" doc rather than in the "Get started with InfluxDB tasks" doc.

Thanks, I debated that. As it turns out, this might be the only kind of task going forward. In any case, this PR was premature and I'm waiting to hear more.
